### PR TITLE
Fix: compatibility with multicompany transversal mode and more security issue

### PR DESCRIPTION
**Pull request by hregis**

### CODE REVIEW

File: htdocs/societe/commerciaux.php [hunk 1]
```diff
@@ -61,17 +61,17 @@
 
 		if (empty($reshook)) $soc->add_commercial($user, $_GET["commid"]);
 
-		header("Location: commerciaux.php?socid=".$soc->id);
+		header("Location: ".$_SERVER["PHP_SELF"]."?socid=".$soc->id);
```
Comment (marcosgdf) [inline htdocs/societe/commerciaux.php:64]: Is this OK? Looks like the page is missing

Comment (hregis) [inline htdocs/societe/commerciaux.php:64]: i don't understand... What is the problem ?

Comment (marcosgdf) [inline htdocs/societe/commerciaux.php:64]: Why building urls in a different way than the rest of Dolibarr?

Comment (hregis) [inline htdocs/societe/commerciaux.php:64]: $_SERVER["PHP_SELF"] has always been used when the URL is on the same file

Comment (marcosgdf) [inline htdocs/societe/commerciaux.php:64]: Oh :ok_hand:

**Pull request merged**